webview-ui/src/components/chat/__tests__/Announcement.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import { render, screen, fireEvent } from "@testing-library/react"
+import { describe, it, expect, vi } from "vitest"
+import Announcement from "../Announcement"
+
+vi.mock("react-i18next", () => ({
+	useTranslation: () => ({
+		t: (key: string, passed: object | undefined) => {
```

Comment:
.version does not exist on object
lets just use any
```suggestion
		t: (key: string, passed: any) => {
```